packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -272,15 +273,14 @@ const getListTerms = ({ tagId, sortedBy, limit, after }: { tagId: string; sorted
   algoDecayFactorFastest: 3.0,
 });
 
-const formatDollars = (amount: number) => "$" + formatStat(amount);
+const SHOW_AMOUNT_RAISED = true;
```

Comment:
Change this to `true` to see the amounts. When this is deployed, I'll just set all the ids straight after and then force merge a followup to remove this (I didn't want to add an extra setting)